API/urlfrontier.proto

```diff
@@ -290,3 +296,13 @@ message CrawlLimitParams {
   // crawl ID
   string crawlID = 3;
 }
+
+message URLStatusRequest {
+    /** URL for which we request info */
+    string url = 1;
+    /** ID for the queue **/
+    string key = 2;
```

Comment:
The key can be empty / not set when calling [PutURLs](https://github.com/crawler-commons/url-frontier/blob/master/API/urlfrontier.proto#L236), in which case the backend implementation works out a sensible value. Will the behaviour be the same when calling **GetURLStatus** ?

Reply:
Yes, I changed it so that the behaviours are aligned